syntax = "proto3";

package keto.proto;

/**
 * This is an example of a protobuf message.
 */
message CheckForAccount {
    uint32 version = 1;
    bytes account_hash = 2;
    bool found = 3;
}

message AccountInfo {
    uint32 version = 1;
    bytes account_hash = 2;
    bytes parent_account_hash = 3;
    string account_type = 4;
    string graph_name = 5;
}

message ClearDirtyCache {
    uint32 version = 1;
}



